vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.jsp

```diff
@@ -44,6 +44,97 @@ function uploadImage() {
 } 
 	
 $(document).ready(function() {
+	//-------- edit description --------
+	$("#submitDescription").hide()
+	$("#editDescription").click(function() {
+		var val = $("#description").val();
+		$("#description").html(function (i, val) {
+	        return '<textarea id="newDescription" style="width:250%; height:200%;" class="form-control" type="text">'+val+'</textarea>';
+	    });
+		$("#editDescription").hide()
+		$("#submitDescription").show()
+		
+	});
+	
+	$("#submitDescription").click(function() {
+		var formData = new FormData();
+		formData.append('description', $("#newDescription").val());
+		$.ajax({
+		url: "<c:url value="/staff/module/${module.id}/slide/${slide.id}/edit/description?${_csrf.parameterName}=${_csrf.token}" />",
+		type: 'POST',
+		cache       : false,
+		contentType : false,
+		processData : false,
+		data: formData,
+		enctype: 'multipart/form-data',
+		success: function(data) {
+			// replace text box with new description
+			$("#submitDescription").hide()
+			$("#editDescription").show()
+			var val = $("#newDescription").val();
+			$("#newDescription").remove()
+			$("#description").text(val)
+			
+		},
+		error: function(data) {
+			console.log(data);
+			var alert = $('<div class="alert alert-danger alert-dismissible fade show" role="alert"><p>We are sorry but something went wrong. Please try again later.</p><button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button></div>');
+			$('.error').append(alert); 
+			$(".error").delay(4000).slideUp(500, function(){
+			    $(".error").empty();
+			});
```

Comment:
does this fade the alert out? I would leave it there for the user to close. If the user submits a change and then turns around to talk to someone or leave for a little while and something goes wrong, they might miss the error alert.